### PR TITLE
[WIP] Enabling the 'snapshot' feature through OCS Operator

### PR DESCRIPTION
**Pull request by aruniiird**
Signed-off-by: Arun Kumar Mohan <amohan@redhat.com>

### CODE REVIEW

**Comment by aruniiird** [comment]
@Madhu-1  , I have enabled `CSI_ENABLE_SNAPSHOTTER` to true in OCS Operator generated CSV, in order to enable snap shot feature. Currently labelled the PR as WIP to see if we need additional changes in OCS Operator front.

File: deploy/olm-catalog/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1904,7 +1904,7 @@ spec:
                     fieldRef:
                       fieldPath: metadata.namespace
                 - name: CSI_ENABLE_SNAPSHOTTER
-                  value: "false"
+                  value: "true"
```
Comment (Madhu-1) [inline deploy/olm-catalog/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml:1907]: @aruniiird Thanks for the PR, we have provided this option to disable snapshot as it was in alpha version and it's going to be the default. I thought to remove this flag in the next ROOK version as it is going to be backward compatible and enabled always in OCS.

@travisn Any suggestion?

 apart from this change, we need to create and delete snashotclass for both cephfs and rbd.

Comment (travisn) [reply deploy/olm-catalog/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml:1907]: @Madhu-1 Per discussion in the csi meeting, agreed that we can just remove this setting from rook and always have the snapshotter enabled. Nobody outside OCS was using that feature to disable it.

**Comment by aruniiird** [comment]
According to the above comments, this PR is no longer needed...
Closing this PR...

**Pull request closed without merging**